app/controllers/games.rb

```diff
@@ -4,6 +4,7 @@
 end
 
 post '/games' do
+  #ZM: SEE PR
   userid = User.find_by(id: session[:user_id]).id
```

Comment:
You should have a helper method called current_user that will do this same operation.